src/lib/apiErrorHandler.js

```diff
@@ -107,6 +106,10 @@ export const handleError = async (req, res, error, logContext) => {
 export function handleUnexpectedErrorInCatchBlock(req, res, error, logContext) {
   logger.error(`${logContext.namespace} - Unexpected error in catch block - ${error}`);
   res.status(INTERNAL_SERVER_ERROR).end();
+  if (error instanceof Sequelize.ConnectionAcquireTimeoutError) {
+    logger.error(`${logContext.namespace} - Critical error: Restarting server.`);
```

Comment:
Please make the error message state that this is the Sequelize.ConnectionAcquireTimeoutError that is the critical error